packages/core/data/mm/world/beneath_the_well.yml

```diff
@@ -7,19 +7,19 @@
 "Beneath The Well North Section":
   dungeon: BtW
   events:
-    WELL_HOT_WATER: "event(FISH) && (has_explosives || has_mask_zora || trick_keg_explosives || trick(MM_WELL_HSW))"
-    WATER: "true"
-    FISH: "true"
-    BUGS: "event(WATER) && (can_use_fire_arrows || has_sticks)"
-    BOMBS: "event(WATER) && (can_use_fire_arrows || has_sticks)"
-    RUPEES: "event(WATER) && (can_fight || has_weapon_range || has_explosives)"
+    WELL_HOT_WATER: "has_bottle && event(FISH) && (has_explosives || has_mask_zora || trick_keg_explosives || trick(MM_WELL_HSW))"
```

Comment:
has_bottle isn't really needed here since all FISH events check for bottle regardless.

Reply:
Fixed